_posts/2018-11-20-developer-view-oozie-vs-airflow.md

```diff
@@ -25,15 +25,16 @@ These are some of the scenarios for which built-in code is available in the tool
 * Add dependency checks; for example triggering a job if a file exists, or triggering one job after the completion of another.
 * Trigger a job from a failure step.
 * Cause the job to timeout when a dependency is not available.
-* Add Standard Level Agreement (SLA) to jobs.
+* Add Service Level Agreement (SLA) to jobs.
 
 With cron, you have to write code for the above functionality, whereas Oozie and Airflow provide it.
 
 
 ### Oozie ###
 
 Apache Oozie is a workflow scheduler which uses Directed Acyclic Graphs (DAG) to schedule Map Reduce Jobs (e.g. Pig, Hive, Sqoop, Distcp, Java functions). 
```

Comment:
I think this would read more smoothly if you added the link to the first mention of Apache Oozie, rather than repeating the name in the second sentence (so keep the text below as "It's an open source project").

Same comment for Airflow below.